src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv401.cql

```diff
@@ -718,8 +720,8 @@ define ReportEncounterRiskFactorsForOpioidRelatedHarms:
   return GetEncounterDiagnosesRiskFactorsForOpioidRelatedHarms(E))) D
   sort by Visit.Start desc, Visit.End desc
 
-// Shape the pregnancy Observation as a Condition, since this is the backup approach for detecting a pregnancy in case
-// there are no active Conditions found.
+//Shape the pregnancy Observation as a Condition, since this is the backup approach for detecting a pregnancy in case
+//there are no active Conditions found.
```

Comment:
```suggestion
// Shape the pregnancy Observation as a Condition, since this is the backup approach for detecting a pregnancy in case
// there are no active Conditions found.
```